scripts/uberenv/packages/raja/package.py

```diff
@@ -225,6 +225,12 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
                 cfg.write(cmake_cache_entry("BLT_CMAKE_IMPLICIT_LINK_DIRECTORIES_EXCLUDE",
                 "/usr/tce/packages/gcc/gcc-4.9.3/lib64;/usr/tce/packages/gcc/gcc-4.9.3/gnu/lib64/gcc/powerpc64le-unknown-linux-gnu/4.9.3;/usr/tce/packages/gcc/gcc-4.9.3/gnu/lib64;/usr/tce/packages/gcc/gcc-4.9.3/lib64/gcc/x86_64-unknown-linux-gnu/4.9.3"))
 
+        if ("%xl" in spec) and ("+cuda" not in spec):
```

Comment:
@kab163 @davidbeckingsale Is this ok with you? Hopefully this doesn't interfere with any ongoing XL+Cuda efforts.

Reply:
I think you can put this (lines 228-232) in the "else" part of the "if +cuda in spec" statement below. (And just have a if ("xl" in spec) instead). Also, I think you can get rid of these flags in the spec command that you have in lassen-jobs.yml (since they're duplicated).

Reply:
In terms of XL+Cuda, we are kind of on-hold with that until we can figure out a subset of tests to use (to decrease linking times). So getting this to work first will be good.

Reply:
> I think you can put this (lines 228-232) in the "else" part of the "if +cuda in spec" statement below. (And just have a if ("xl" in spec) instead).

Good catch, done.

> Also, I think you can get rid of these flags in the spec command that you have in lassen-jobs.yml (since they're duplicated).

When I tried that, these flags were not processed. I think this is because the cxxflags in the spec line override the logic I added, and just use whatever Spack specifies (with the overridden cxxflags).